<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pro_Cancel</class>
 <widget class="QDialog" name="Pro_Cancel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_times">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>[60]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>原流水号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1_oldTrace">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>276</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>消费撤销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>276</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2_pwd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>145</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入密码：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="line0_card">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelHint">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卡号信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请刷卡&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="transRst">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>173</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line0_card</tabstop>
  <tabstop>line1_oldTrace</tabstop>
  <tabstop>line2_pwd</tabstop>
  <tabstop>transRst</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>confirmButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
